test/external-bundle/ajv.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { assert } from "https://deno.land/std@0.170.0/testing/asserts.ts";
+
+import * as ajv from 'http://localhost:8080/ajv@8.12.0?bundle&external=fast-deep-equal'
+
+function use(arg: any) {
```

Comment:
what this function here for?

Reply:
added comment. It's to avoid unused import removal

Reply:
ooh, makes sense